backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisWebservice.java

```diff
@@ -52,6 +52,8 @@ public static void start(int port) {
      * WARNING !! SPARK JAVA USES DESIGN WHERE THE ORDER OF REQUEST DEFINITION DETERMINES THE HANDLER
      */
 
+    post("/:schema/api/email/*", ACCEPT_JSON, EmailApi::send);
```

Comment:
unexpected to do this here, instead of in a ContactApi (please don't call it EmailApi)